build.gradle.kts

```diff
@@ -60,11 +64,8 @@ dependencyCheck {
 
 publishing {
     publications {
-        register<MavenPublication>(rootProject.name) {
-            groupId = project.group as? String
-            artifactId = project.name
-            version = project.version as? String
-            from(components["java"])
```

Comment:
I guess this `from` removal has an effect of this change in the POM
![image](https://user-images.githubusercontent.com/2906988/201527283-75fc72d7-cadc-4619-9237-e294b08b8c38.png)

Reply:
ah, ok, so I was diffing the original POM with the common pom... if I diff the original with the `-jvm` pom, it looks good.

Reply:
changed to dependency `implementation`